L1Trigger/L1TCaloLayer1/src/L1TCaloLayer1FetchLUTs.hh

```diff
@@ -2,6 +2,7 @@
 #define L1TCaloLayer1FetchLUTs_hh
 
 #include "UCTGeometry.hh"
+#include "FWCore/Framework/interface/EventSetup.h"
```

Comment:
Seems that vector and array should also be included.

Reply:
I've included them, but since the test didn't complained I'd guess UCTGeometry already does.